its/src/test/java/com/sonar/it/csharp/AnalysisWarningsTest.java

```diff
@@ -59,6 +59,6 @@ public void analysisWarningsImport() throws IOException {
 
     Ce.Task task = TestUtils.getAnalysisWarningsTask(ORCHESTRATOR, buildResult);
     assertThat(task.getStatus()).isEqualTo(Ce.TaskStatus.SUCCESS);
-    assertThat(task.getWarningsList()).containsExactly("First message", "Second message");
+    assertThat(task.getWarningsList()).containsExactly("First message", "Second message", "BROKEN IT");
```

Comment:
Do you know why the lists on AZP shows everything twice?
The stats on TOP seem correct, it's just the list of tests below that is weird
![image](https://user-images.githubusercontent.com/57188685/183061144-9a61511a-d520-4662-94a0-ed85a15de164.png)

Reply:
Is it result of some test runs?

Reply:
That was my fault. One of the steps of the CI run was canceled ("machine shutdown") and I pressed "re-run failed jobs". The tests were reported a second time, therefore.